<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QCheckBox" name="check_active">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Hysteresis</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_interval">
          <property name="text">
           <string>Interval</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTimeEdit" name="time_hysteresis">
          <property name="maximumTime">
           <time>
            <hour>0</hour>
            <minute>10</minute>
            <second>0</second>
           </time>
          </property>
          <property name="minimumTime">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>5</second>
           </time>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::MinuteSection</enum>
          </property>
          <property name="displayFormat">
           <string>mm:ss</string>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>1</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTimeEdit" name="time_interval">
          <property name="minimumTime">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>1</second>
           </time>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::HourSection</enum>
          </property>
          <property name="displayFormat">
           <string>HH:mm:ss</string>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>5</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="line_message"/>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_clock">
          <property name="text">
           <string>Alarm</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTimeEdit" name="time_alarm">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>HH:mm:ss</string>
          </property>
          <property name="time">
           <time>
            <hour>8</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="check_repeat">
        <property name="text">
         <string>Repeat</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Repeat Times</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_repeat">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Repeat times, -1 = forever</string>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Audio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="box_audio"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_remain">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_confirm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Confirm Remind</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_repeat</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_repeat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_active</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_active</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_save</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_interval</sender>
   <signal>toggled(bool)</signal>
   <receiver>time_interval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_clock</sender>
   <signal>toggled(bool)</signal>
   <receiver>time_alarm</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
